### ISSUE TITLE
CVC Dataset Train

### ISSUE THREAD

**Issue by lihui1998** [sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection#12]
Dear, thank you very much for the dataset.py file provided, but we still can't train on the CVC dataset because of the lack of something like kaist_annotations_test20.json, test-all-20.txt, train-all-02.txt, config.py , could you please provide these documents again, thank you.

**Comment by socome** [comment]
Thank you for your interest in our work again! Unfortunately, We don't have the  config file for training the CVC-14 datasets because we only organize our code for training the KAIST datasets with config file. However, I think you can easily change the released config file.

[CVC14.zip](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/files/8055624/CVC14.zip)

**Comment by socome** [comment]
Did you change the evaluation code properly? I mean some part of the evaluation code in this repo are hardcoded like [this](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/d87d96b24ba3148a4c983ae0f30476d0725d406f/evaluation_script/evaluation_script.py#L579). Also, We follow the protocol introduced in [21] as other studies also adopt this. Thus, we only uses bounding boxes in grey images for training and testing without multi-label learning(we have a pointed out in the paper).

[21] K. Park, S. Kim, and K. Sohn, “Unified multi-spectral pedestrian detection based on probabilistic fusion networks,” Pattern Recognit., vol. 80, pp. 143–155, 2018

**Comment by lihui1998** [comment]
Thank you for your reply, since we are very interested in your work, we hope to reproduce the data in the paper. Based on your previous suggestion, our issue remains unresolved. Below I describe the work we do in detail, and hope to get your suggestions. We first resized the images on the basis of the original dataset, unified to 640×471, and then we merged the contents of the FramesNeg and FramesPos folders in the training set into the Frames file. After that we trained using the datasets.py, CVC-14_test_all_Full.json, Images_train_all.txt and Images_test_all.txt you provided, since we still used the config.py file for the kaist dataset, we did the datasets.py file Modified as follows;
![2ed74a4523146f5bebd977c1b0bf062](https://user-images.githubusercontent.com/51204863/154295960-5108e1ea-246b-43b6-a8f9-dbd0fe1f6196.png)
In addition, we also modified some details in the evaluation code;
![2ed74a4523146f5bebd977c1b0bf062](https://user-images.githubusercontent.com/51204863/154296026-cbb467a0-77e6-479a-ab87-59dc4e5833d2.png)
Although it can be trained, it can't go down during the day as described in the previous question. In addition, according to your suggestion, we only use the bounding box in the gray image for training and testing, and no longer convert the image in visible to RGB during training. , but the effect is worse. Can you give further advice on our work? Thank you so much, looking forward to your reply!

**Comment by lihui1998** [comment]
![9812c5ec465b4257bb9af159445dadf](https://user-images.githubusercontent.com/51204863/154296302-39d5c997-ab31-430a-a479-3296db8ac267.png)
Sorry, this is what was modified in the eval code

**Comment by chiyongke** [comment]
> Thank you for your answer. We successfully trained the cvc-14 dataset using the files you provided. The image size we uniformly used is 640*471, but the error rate of the dataset during the day is as high as 50%, which cannot achieve the effect of the paper. Can you give us some advice. thanks. <img alt="af4c9392726349e7a1dcd835b41d5cb" width="283" src="https://user-images.githubusercontent.com/51204863/154106159-e3cc8aa3-733f-4417-884a-edc5f8a28931.png">

Hello, how can you modify the config.py, datasets.py file of CVC-14, I always report the following errors after modification：assert n_priors == predicted_locs.size(1) == predicted_scores.size(1)

**Comment by xw1508** [comment]
> Thank you for your answer. We successfully trained the cvc-14 dataset using the files you provided. The image size we uniformly used is 640*471, but the error rate of the dataset during the day is as high as 50%, which cannot achieve the effect of the paper. Can you give us some advice. thanks. <img alt="af4c9392726349e7a1dcd835b41d5cb" width="283" src="https://user-images.githubusercontent.com/51204863/154106159-e3cc8aa3-733f-4417-884a-edc5f8a28931.png">

Hello, may I ask if you could share the training and validation program for CVC-14? Due to limited abilities, it is not possible to modify the code well. Your work is very useful. Thank you very much
